README.md

```diff
@@ -421,11 +249,15 @@ Now you should be able to run Julia like this:
 
 On Windows, double-click `usr/bin/julia.exe`.
 
-If everything works correctly, you will see a Julia banner and an interactive prompt into which you can enter expressions for evaluation.
-You can read about [getting started](https://julialang.org/manual/getting-started) in the manual.
+If everything works correctly, you will see a Julia banner and an
+interactive prompt into which you can enter expressions for
+evaluation.  You can read about [getting
+started](https://julialang.org/manual/getting-started) in the manual.
 
-**Note**: Although some system package managers provide Julia, such installations are neither maintained nor endorsed by the Julia project. They may be outdated
-and/or unmaintained. We recommend you use the official Julia binaries instead.
+**Note**: Although some system package managers provide Julia, such
+installations are neither maintained nor endorsed by the Julia
+project. They may be outdated and/or unmaintained. We recommend you
+use the official Julia binaries instead.
 
 ## Editor and Terminal Setup
```

Comment:
Is this section important to keep? Seems like it could also be moved.

Reply:
This is important for the README. Will move.

Reply:
I feel it could move to the manual.

Reply:
Same

Reply:
Actually, as I go through all these - I do want to leave this warning here.

Reply:
Leaving this in place for now. Can eventually go into the manual - but I am not doing moves to the manual in this PR.